ecommers/lib/ui/pages/checkout_page.dart

```diff
@@ -268,8 +270,10 @@ class _CheckoutPageState extends State<CheckoutPage> {
   Widget _buildRowAction({String imagePath, Text text}) {
     return Row(
       children: <Widget>[
-        Image.asset(
-          imagePath,
+        CachedNetworkImage(
```

Comment:
as I see there are a lot of duplications:
          errorWidget: (context, url, error) =>
              SvgPicture.asset(Assets.warningIcon),
          fit: BoxFit.scaleDown, 
can we create widget to refactor this?

Reply:
Sure :)